src/backend/metal/src/lib.rs

```diff
@@ -418,6 +428,16 @@ impl Instance {
             layer
         };
 
+        // Set set top-left layer placement, so that resizing the window
+        // doesn't result in stretching, which combined with newly rendered
+        // frames results in jerky movement.
+        #[allow(non_upper_case_globals)]
+        const NSViewLayerContentsPlacementTopLeft: i32 = 11;
```

Comment:
I'm a bit concerned about this path. The view is something controlled by the client, not us. We should try to do minimum changes, if any, to that view. Is it reasonable to say that this should be a `winit` problem? Or basically, whoever actually manages the `NSView`, while gfx-rs shouldn't be that person.

Reply:
now, I agree that winit should be setting the placement, but last time I tried, it didn't work, so I figured it is something about the layer within the view, and not the view itself.

I can do more tests with winit setting it too, but it'll take some time until I can get to it

Reply:
thanks! if there is something we are missing here, we'd need to know, what it is.
Note that this issue doesn't need to block gfx-0.8, since it's not an API breaking fix.